survey-sdk/src/main/java/co/nimblehq/survey/sdk/request/BaseRequest.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.survey.sdk.request
+
+import com.squareup.moshi.Json
+
+open class BaseRequest {
```

Comment:
from the design concept, base component should be abstract

Reply:
@luongvo Agree with this.